src/lib.rs

```diff
@@ -3334,6 +3334,24 @@ impl Build {
                     // Use the GNU-variant to match other Unix systems.
                     name = format!("g{}", tool).into();
                     self.cmd(&name)
+                } else if target.os == "vxworks" {
+                    name = format!("wr-{}", tool).into();
+                    self.cmd(&name)
+                } else if target.os == "nto" {
+                    // Ref: https://www.qnx.com/developers/docs/8.0/com.qnx.doc.neutrino.utilities/topic/a/ar.html
+                    name = match target.arch {
+                        "i586" => format!("ntox86-{}", tool).into(),
```

Comment:
Maybe I wasn't clear here? `target.arch` cannot be `i586`, we map that away in the target translation / with the info from Cargo. If you want to match on `i586-...` in target name, use `target.full_arch`.
```suggestion
```

Reply:
Same above with `-Vgcc_ntox86_cxx` I think?

Reply:
I wonder how did it pass the test I performed. :thinking: I will change it quickly.